e2e/support/helpers/e2e-qa-databases-helpers.js

```diff
@@ -104,13 +104,21 @@ function assertOnDatabaseMetadata(engine) {
 
 function recursiveCheck(id, i = 0) {
   // Let's not wait more than 20s for the sync to finish
-  if (i === 40) {
-    throw new Error("The sync is taking too long. Something is wrong.");
+  if (i === 20) {
+    cy.task(
+      "log",
+      "The DB sync isn't complete yet, but let's be optimistic about it",
+    );
+    return;
```

Comment:
So I think the solution here is just, don't throw an error if the sync doesn't report as complete after 20 seconds. My investigation revealed that often the sync actually _is_ complete according to the BE console output, even though the API says it isn't.

**Likely case**: the sync is actually complete and everything is fine.
**Fine case**: the sync failed and this workflow doesn't even use this DB, so we didn't block it unnecessarily.
**Worst case**: the sync isn't complete and a test fails, which is what's happening already, so we're no worse off.

Reply:
Worth trying 🤷‍♂️